hack/generate_addons.sh

```diff
@@ -19,10 +20,11 @@ kustomize_build() {
 
 helm_template eks aws-node-termination-handler 0.13.3
 helm_template autoscaler cluster-autoscaler 1.0.4 -chart
-helm_template nvdp nvidia-device-plugin 0.8.2
+helm_template nvdp nvidia-device-plugin 0.7.0
+helm_template eks aws-load-balancer-controller 1.1.0
```

Comment:
why `1.1.0` and not the current `1.1.5`? Afterall, that is the version defined in the gitRef we're getting the CRDs from below.